opm/output/eclipse/AggregateConnectionData.cpp

```diff
@@ -75,18 +75,31 @@ namespace {
             : &grid;
 
         std::size_t connID = 0;
+        bool skip_connection = false;
+        int connection_counter = 0;
+        std::string last_connection_lgr_tag = "";
         for (const auto* connPtr : well.getConnections().output(*lgrid)) {
             if (connPtr->kind() == Opm::Connection::CTFKind::DynamicFracturing) {
                 // Don't emit, or count, connections created by dynamic fracturing.
                 continue;
             }
+            std::string current_lgr_lgr_tag = well.get_lgr_well_tag().value_or("");
+
+            if (well.is_lgr_well()) {
+                if ((current_lgr_lgr_tag == last_connection_lgr_tag) and (connection_counter > 0)) {
+                    // After the first connection of a LGR well, subsequent connections of the same well are skipped.
+                    skip_connection = true;
+                }
+            }
 
             const auto* dynConnRes = (wellRes == nullptr)
                 ? nullptr : wellRes->find_connection(connPtr->global_index());
 
             connOp(wellName, wellID, isProd, *connPtr, connID,
-                   connPtr->global_index(), dynConnRes);
-
+                   connPtr->global_index(), skip_connection, dynConnRes);
```

Comment:
Why does the `connOp` need to know about the `skip_connection` flag?  If we're really supposed to skip the connection, then just don't call the `connOp()` in the first place.